src/pages/home/HeaderView.js

```diff
@@ -113,7 +113,7 @@ const HeaderView = (props) => {
                         </Pressable>
                     </Tooltip>
                 )}
-                {props.report && props.report.reportName && (
+                {Boolean(props.report && props.report.reportName) && (
```

Comment:
I added this to fix a RN error that I was seeing where text is a child of View

Reply:
I haven't seen it done like this, good thinking.